unit-10/main.js

```diff
@@ -0,0 +1,132 @@
+// bài 1: In một bảng nhân của một số bất kỳ với các số từ 1 tới 10 và hiển thị kết quả.
+const tableOfMultipliers = num => {
+  for (let i = 1; i <= 10; i++) {
+    console.log(num * i)
+  }
+}
+tableOfMultipliers(10)
+
+/* bài 2: Hàm nhận tham số là n (số nguyên dương),
+  1 <= n <= 30 In các số chẵn từ 1 đến n Vd n = 6: in ra: 2 4 6 */
+const getEven = num => {
+  if (1 <= num && num <= 30) {
+    for (let i = 1; i <= num; i++) {
+      if (i % 2 !== 0) continue
+      console.log(i)
+    }
+  }
+}
+getEven(30)
+
+/* bài 3: Hàm nhận tham số là n (số nguyên dương),
+  1 <= n <= 30 Tính tổng từ 1 đến n
+  vd nhập vào 6: Tổng = 1 + 2 + 3 + 4 + 5 + 6 = 21  */
+const sum = num => {
+  if (1 <= num && num <= 30) {
+    let sumIndex = 0
+    for (let i = 1; i <= num; i++) {
+      sumIndex += i
+    }
+    console.log(sumIndex)
+  }
+}
+sum(10)
+
+/* bài 4: Hàm nhận tham số là n (số nguyên dương),
+  1 <= n <= 30 In ra n! (giai thừa của n)  */
+const factorialCalculate = num => {
+  if (1 <= num && num <= 30) {
+    let productMultiplier = 1
+    for (let i = 1; i <= num; i++) {
+      productMultiplier *= i
+    }
+    console.log(productMultiplier)
+  }
+}
+factorialCalculate(8)
+
+/* bài 5: Hàm nhận vào 1 mảng đếm xem trong mảng có bao nhiêu số chẵn
+  Vd: param là [1, 2, 3, 6, 2, 9, 0] thì return 4 vì có 4 số chẵn  */
+const getEven = arr => {
+  let count = 0
+  arr.forEach( num => {
+    if (num % 2 === 0) {
+      count++
+    }
+  })
+  console.log(count)
+}
+getEven ([2, -2, 8, 6, 1, 0, 0])
+
+/* bài 6: Hàm nhận vào 1 mảng chứa các chữ cái bị trùng lặp ngẫu nhiên
+  Tính toán và trả về 1 mảng đã xóa đi các chữ cái bị trùng lặp  */
+const similarDelete = arr => {
+  const arrLetter = arr.filter( (letter, index) => {
```

Comment:
```suggestion
  const arrLetter = arr.filter((letter, index) => {
```
Giải bài này bằng tay, không dùng `Set`, `filter`, ...

Tham khảo: https://github.com/daiphung1998/Baitapvenha_REACTJS_2009E/pull/2/files#r511375693